modules/opentelemetry-otlp-http-exporter/src/test/scala/io/janstenpickle/trace4cats/opentelemetry/otlp/OpenTelemetryOtlpHttpSpanCompleterSpec.scala

```diff
@@ -21,10 +23,18 @@ class OpenTelemetryOtlpHttpSpanCompleterSpec extends BaseJaegerSpec {
       }
     )
     val batch = Batch(Chunk(updatedSpan.build(process)))
+    val completer = BlazeClientBuilder[IO](global).resource.flatMap { client =>
```

Comment:
I published v0.12.0+42-5e4d5380 of trace4cats-jaeger-integration-test.
Please use the new API for BlazeClientBuilder.